ci-operator/step-registry/mirror-images/by-oc-adm/mirror-images-by-oc-adm-ref.metadata.json

```diff
@@ -0,0 +1,13 @@
+{
+	"path": "mirror-images/by-oc-adm/mirror-images-by-oc-adm-ref.yaml",
```

Comment:
How about "mirror-release-image/mirror-release-image-ref.yaml" instead? I don't think the sub-directory "by-oc-adm" is necessary.

Reply:
there is another tool, called `oc-mirror`, I think we also need to integrate it into Prow CI in the future, so at that time, it will be implemented in `mirror-images/by-oc-mirror`

Reply:
But if we are not the QE in charge of testing the tool, we can use any, right? If so, we could update our script if we decide to use the new tool.

Reply:
we might use both of them, and these steps could be also shared with other teams.

Reply:
Don't quite understand, anyway, currently name it as "**mirror-release-image/mirror-release-image-ref.yaml**", and when we need to add for "oc-mirror" add the sub-folder "oc-mirror" under "mirror-release-image" (e.g. "mirror-release-image/oc-mirror/mirror-release-image-oc-mirror-ref.yam").

Reply:
Seem like currently is managing mirror approaches by folder name.

Reply:
Yes, the folder name indicates the way we use.

> Don't quite understand, anyway, currently name it as "**mirror-release-image/mirror-release-image-ref.yaml**", and when we need to add for "oc-mirror" add the sub-folder "oc-mirror" under "mirror-release-image" (e.g. "mirror-release-image/oc-mirror/mirror-release-image-oc-mirror-ref.yam").

@jianli-wei you mean using `oc adm` as the default way (`mirror-release-image`), and `oc-mirror` will be another option `mirror-release-image-oc-mirror`, is it right? 
It does an option, personally, I prefer to set it explicitly, like `mirror-release-image-oc-mirror` and `mirror-release-image-oc-adm`, but both of above ways are ok fo me, @jinyunma @jianlinliu wdyt?

Reply:
I am okay with the present folder naming.